hawkular-nest/hawkular-nest-itest-parent/hawkular-nest-itest/src/test/resources/arquillian.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2015 Red Hat, Inc. and/or its affiliates
+    and other contributors as indicated by the @author tags.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<arquillian xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns="http://jboss.org/schema/arquillian"
+            xsi:schemaLocation="http://jboss.org/schema/arquillian
+            http://jboss.org/schema/arquillian/arquillian_1_0.xsd">
+  <container qualifier="default" default="true">
+    <configuration>
+      <property name="jbossHome">${jbossHome}</property>
```

Comment:
@jpkrohling , yes, I could get rid of `arquillian.xml` e86fda4 but the proper property name is `jboss.home` see https://github.com/hawkular/hawkular-commons/commit/e86fda44d33ec954ddcb722c4ba89ac1127f18f3#diff-5c22a8c7340bc3119e2ecaee92e4d375R173